src/SFA.DAS.DATA.MANAGEMENT/Tables/Stg_GA_SessionDataDetail.sql

```diff
@@ -0,0 +1,42 @@
+﻿CREATE TABLE [Stg].[GA_SessionDataDetail]
+(
+		[GASD_Id] BIGINT IDENTITY(1,1)					NOT NULL,
+		[FullVisitorId]									[NVarchar](500) NULL,
+		[ClientId]										[NVarchar](500) NULL,			
+		[VisitId]										[NVarchar](500) NULL,
+		[VisitNumber]									[NVarchar](500) NULL,
+		[VisitStartDateTime]							[NVarchar](500) NULL,
+		[VisitDate]										[NVarchar](500) NULL,
+		[VisitorId]										[NVarchar](500) NULL,
+		[UserId]										[NVarchar](500) NULL,		
+		[Hits_Page_PagePath]							[NVarchar](Max) NULL,
+		[Hits_Page_PagePathLevel1]						[NVarchar](Max) NULL,
+		[Hits_Page_PagePathLevel2]						[NVarchar](Max) NULL,
+		[Hits_Page_PagePathLevel3]						[NVarchar](Max) NULL,
+		[Hits_Page_PagePathLevel4]						[NVarchar](Max) NULL,
+		[Hits_Time]										[NVarchar](500) NULL,		
+		[Hits_IsEntrance]								[NVarchar](500) NULL,
+		[Hits_IsExit]									[NVarchar](500) NULL,
+		[EmployerId]									[NVarchar](500) NULL,
+		[ID2]											[NVarchar](500) NULL,
+		[ID3]											[NVarchar](500) NULL,
+		[ESFAToken]										[NVarchar](500) NULL,		
+		[EventCategory]									[NVarchar](max) NULL,
+		[EventAction]									[NVarchar](max) NULL,
+		[EventLabel]									[NVarchar](max) NULL,
+		[EventLabel_ESFAToken]							[NVarchar](500) NULL,		
+		[EventLabel_Keyword]							[NVarchar](500) NULL,
+		[EventLabel_Postcode]							[NVarchar](500) NULL,
+		[EventLabel_WithinDistance]						[NVarchar](500) NULL,
+		[EventLabel_Level]								[NVarchar](500) NULL,
+		[EventValue]									[NVarchar](Max) NULL,
+		[CD_ClientId]									[NVarchar](500) NULL, 
+		[CD_SearchTerms]								[NVarchar](500) NULL, 
+		[CD_UserId]										[NVarchar](500) NULL, 
+		[CD_LevyFlag]									[NVarchar](500) NULL, 
+		[CD_EmployerId]									[NVarchar](500) NULL, 
+		[CD_ESFAToken]									[NVarchar](500) NULL, 
+		[CD_LegalEntityId]								[NVarchar](500) NULL,
+		[FileName]										[Nvarchar](500) NULL,
+		[StgImportDate]									[Nvarchar](500) NULL
```

Comment:
Q) Is it ok for StgImportDate to be Nvarchar(500)?
Q) Don't we have to assign the table to Beta User Role?

Reply:
New changes done on this looks good.